### ISSUE TITLE
java.net.UnknownHostException: Unable to resolve host "api.tapglue.com":

### ISSUE THREAD

**Issue by ThathaPavanNagaraju** [tapglue/android_sdk#55]
10-29 18:41:31.524 26707-26790/com.tapglue.example D/OkHttp: --> POST https://api.tapglue.com/0.4/users http/1.1
10-29 18:41:31.524 26707-26790/com.tapglue.example D/OkHttp: Content-Type: application/json; charset=UTF-8
10-29 18:41:31.524 26707-26790/com.tapglue.example D/OkHttp: Content-Length: 174
10-29 18:41:31.524 26707-26790/com.tapglue.example D/OkHttp: Authorization: Basic MmRhMmQ3OTMzNmMyNzczYzYzMGNlNDZmNWQyNGNiNzY6
10-29 18:41:31.524 26707-26790/com.tapglue.example D/OkHttp: X-Tapglue-OS: Android
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: X-Tapglue-OSVersion: 7.0
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: X-Tapglue-Manufacturer: motorola
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: X-Tapglue-Model: Moto G (4)
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: X-Tapglue-SDKVersion: 3.0.5
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: X-Tapglue-AndroidID: 4cf58a47-7078-49dc-a92e-acba751920b3
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: X-Tapglue-Timezone: Asia/Calcutta
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: {"enabled":true,"followed_count":0,"follower_count":0,"friend_count":0,"is_followed":false,"is_follower":false,"is_friend":false,"password":"supersecret","user_name":"pablo"}
10-29 18:41:31.525 26707-26790/com.tapglue.example D/OkHttp: --> END POST (174-byte body)
10-29 18:41:31.530 26707-26790/com.tapglue.example D/OkHttp: <-- HTTP FAILED: java.net.UnknownHostException: Unable to resolve host "api.tapglue.com": No address associated with hostname

**Comment by xla** [comment]
Would be helpful if you could provide some context to your issue.

Generally Tapglue as a platform is out-of-business since last year. You are able to run it yourself if you can bring the server component up and connect against it.

**Comment by ThathaPavanNagaraju** [comment]
Yeah, how to make the server component up?? I tried many ways to connect from the Android end but couldn't figure it out. could you please help me out. The API response I mentioned was the first API that was hit when sample app is opened.
